styles/style.css

```diff
@@ -63,10 +63,28 @@ p {
 .navigation__list {
   display: flex;
   list-style: none;
+  }
+}
+
+.nav1 {
+  margin: 30px;
+}
+
+ .nav2 {
+  margin: 30px;
+}
+                                        /* added new classes in order to fix navigation spacing issues */
+.nav3 {
+  margin: 30px;
+}
+
+.nav4 {
```

Comment:
In css we can reuse classes in multiple html tags. In this case, I would write only one class like:
```
.nav-item {
    margin: 30px;
}
```
or, because we alreay have a class on all the nav items, I would add `margin` in the existing `.navigation__item` we aleady have:
```
.navigation__item {
     ...
     margin: 30px;
 }
```

In general, in programming we write a lot of code. We want to keep the files as small as possible.